helpers/apiResponse.js

```diff
@@ -1,49 +1,49 @@
 exports.successResponse = function (res, msg) {
-	var data = {
-		status: 1,
-		message: msg
-	};
-	return res.status(200).json(data);
+    var data = {
+        status: 1,
+        message: msg
+    };
+    return res.status(200).json(data);
 };
 
 exports.successResponseWithData = function (res, msg, data) {
-	var resData = {
-		status: 1,
-		message: msg,
-		data: data
-	};
-	return res.status(200).json(resData);
+    var resData = {
+        status: 1,
+        message: msg,
+        data: data
+    };
+    return res.status(200).json(resData);
 };
 
 exports.ErrorResponse = function (res, msg) {
-	var data = {
-		status: 0,
-		message: msg,
-	};
-	return res.status(500).json(data);
+    var data = {
+        status: 0,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 8 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
		status: 0,
```